public/index.html

```diff
@@ -1,36 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="description" content="Construct Hub helps developers find open-source construct libraries for use with AWS CDK, CDK8s, CDKTf and other construct-based tools." />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-<% if (process.env.NODE_ENV === 'development') { %>
-    <meta http-equiv="Content-Security-Policy"
-          content="default-src 'self' https://*.awsstatic.com;
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="description"
+    content="Construct Hub helps developers find open-source construct libraries for use with AWS CDK, CDK8s, CDKTf and other construct-based tools." />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <% if (process.env.NODE_ENV==='development' ) { %>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline' https://*.awsstatic.com;
```

Comment:
The only manual change to this file was adding `unsafe-inline` to `default-src` for the development environment. The rest of the changes were made by `prettier` which is one of our code formatting tools